packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/databases.controller.js

```diff
@@ -0,0 +1,40 @@
+export default class databasesCtrl {
+  /* @ngInject */
+  constructor($anchorScroll, $translate, $q, CucCloudMessage, DatabaseService) {
+    this.$anchorScroll = $anchorScroll;
```

Comment:
Does not seem to be used

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1